### PR TITLE
idp: Utilize ownCloudUserEnabled Attribute to filter users

### PR DESCRIPTION
**Pull request by rhafer**
This adds support for configuring an LDAP Attribute that can be used as a flag to disallow users to login. We currently default to 'ownCloudUserEnabled' as used in the default configuration of the graph service.

### CODE REVIEW

File: changelog/unreleased/enhancement-user-disable.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: allow users to disabled
+
+By setting the 'accountEnabled' property to 'false` for a user via the graph API. Users
```
Comment (mmattel) [inline changelog/unreleased/enhancement-user-disable.md:3]:
```suggestion
By setting the 'accountEnabled' property to `false` for a user via the graph API, users
```

**Comment by mmattel** [comment]
We should add some text to the readme with regards to `LDAP_USER_ENABLED_ATTRIBUTE`.
Currently we have more text in the [ocis admin doku for idm](https://doc.owncloud.com/ocis/next/deployment/services/s-list/idm.html) than here in the readme. I would propose the following: add a section with in the readme `Configuration Hints` same as we have in the admin docs and add a description about the new attribute. After merging, I will complete the text here with the missing stuff in a new PR and transport the changes of the attribute description to the ocis admin docs.

File: changelog/unreleased/enhancement-user-disable.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Enhancement: allow users to disabled
```
Comment (mmattel) [inline changelog/unreleased/enhancement-user-disable.md:1]:
```suggestion
Enhancement: allow users to be disabled
```

**Comment by rhafer** [comment]
> Currently we have more text in the [ocis admin doku for idm](https://doc.owncloud.com/ocis/next/deployment/services/s-list/idm.html) than here in the readme.

This pull request is for the IDP service, not IDM. :slightly_smiling_face: 

IDP has no README at all currently. I will create a separate PR for that.

**Comment by rhafer** [comment]
> Currently we have more text in the [ocis admin doku for idm](https://doc.owncloud.com/ocis/next/deployment/services/s-list/idm.html) than here in the readme.

This pull request is for the IDP service, not IDM. :slightly_smiling_face: 

IDP has no README at all currently. I will create a separate PR for that.

**Review by Excds — APPROVED** [review]
LGTM

**Pull request merged**